styles/_editorjs.scss

```diff
@@ -40,6 +41,7 @@
 
   & .ce-block__content {
     max-width: unset;
+    padding-block: $size3;
```

Comment:
<img width="1429" alt="Screenshot 2023-10-22 at 12 39 15" src="https://github.com/Juratbek/upper_frontend/assets/55176773/3dea6311-adf9-476c-80d1-365df3e45e35">

you are not adding this style in right place

Reply:
padding is very big, 0.3rem is enough

Reply:
Okay, i will fix this

Reply:
@Ayyubxon-coder bu muammo hali ham bor. Siz notog'ri css selector ishlatyabsiz. Sizga kerakli selector qo'shilmagan bo'lsa Samandarga ayting

Reply:
aha xo'p

Reply:
Bro yaxshiku:
<img width="234" alt="image" src="https://github.com/Juratbek/upper_frontend/assets/81279078/be66dedf-345f-4eb9-8c7e-b62cce214058">
<img width="243" alt="image" src="https://github.com/Juratbek/upper_frontend/assets/81279078/c1acea6a-480d-4873-9bb9-fcd29c4f278f">

Reply:
birinchi blokiz paragraph bo'lishi kerak, shuni qilib ko'ring